recipes/ale/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "ale" %}
+{% set version = "0.3.1" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/USGS-Astrogeology/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: fdb399f0943e543b95c51a03942de5b95a6d00b8344a9c849cb363b5d8da73bb 
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}  # [linux]
+    - cmake >=3.10
+    - eigen
+    - gsl
```

Comment:
Libraries that are linked to at runtime need to be in the host section, not the build section. The build section is onyl for build tools, like compilers, cmake etc.